packages/mask/src/plugins/Tips/SNSAdaptor/components/TipsRealmContent/index.tsx

```diff
@@ -11,6 +13,14 @@ const useStyles = makeStyles<{}, 'postTipsButton'>()((theme, _, refs) => ({
         alignItems: 'center',
         color: '#8899a6',
         position: 'relative',
+        [`&.${refs.postTipsButton}::before`]: {
```

Comment:
Is this selector correct? Will `postButtonWrapper` and `postTipsButton` be coexisted?

Reply:
Maybe there's a missing a space.